src/index.html

```diff
@@ -10,13 +10,102 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Логотип"
+        class="logo_img"
+      />
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="./src"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="./src"
```

Comment:
The `href` attribute is set to `./src`, which might not be the correct URL for the 'PHOTO' link. Ensure that this URL points to the intended destination.